Doc/library/stdtypes.rst

```diff
@@ -4303,7 +4305,9 @@ is usually unwise to use them as dictionary keys.)
 
 Dictionaries can be created by placing a comma-separated list of ``key: value``
 pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
-'jack', 4127: 'sjoerd'}``, or by the :class:`dict` constructor.
+'jack', 4127: 'sjoerd'}``, by a dict comprehension, for example: ``{x: x**2 for
+x in iterable}``.  Dictionaries can be created by the :class:`dict` constructor,
```

Comment:
```suggestion
x in iterable}``.  They can also be created by the :class:`dict` constructor,
```

Reply:
The "also" is fine, but I prefer the precise "Dictionaries" over "they".

Reply:
I think "Dictionaries" are more sense.  @rhettinger I made changes. Can you review it again?